### DISCUSSION TITLE
Issue with making an "Allied Military" character.

### DISCUSSION THREAD

**Discussion by Sarcastic216** [bcholmes/StarTrek2d20#16] [General]
Once in the training/academy section for the allied military character you should be able to select 2 disciplines…but it only lets you select 1.

**Comment by bcholmes** [comment]
Some of the tracks for Allied Military require a specific discipline (for example, Intelligence Training requires "Security"). If you try selecting something that isn't "Security", it'll change whatever your current selection is, which might make it look like it's not allowing a second discipline. The required discipline should be in bold, and there should be message under the Disciplines section.

**Comment by Felderburg** [comment]
I'm not sure if it's appropriate to add on something here, but it is related to allied militias. The three named militias (Bajoran, Cardassian, Romulan) still have the regular species options... it seems like the "default" should be the species named, or at least they should have a button on top that corresponds to the named militia before the standard 'select, roll, etc.' buttons.

**Reply by Felderburg** [reply]
Looks like this was added... but now I think that Remans should be an option for Romulan military. I know they're not reaaaally part of the military _per se_, but they were mentioned as being shock troops during the Dominion War.
